lib/philips.js

```diff
@@ -249,4 +659,9 @@ function encodeGradientColors(value, opts) {
 module.exports = {
     decodeGradientColors,
     encodeGradientColors,
+    hueExtend,
+    tzLocal,
+    fzLocal,
```

Comment:
`Local` can be dropped (just like TuYa): https://github.com/Koenkk/zigbee-herdsman-converters/blob/75db090d32280e21caa8c02957488b16faefda31/lib/tuya.js#L1814